files/openmw.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Alexandre Moine <nobrakal@gmail.com> -->
+<application>
+ <id type="desktop">openmw.desktop</id>
+ <metadata_license>CC0-1.0</metadata_license>
```

Comment:
You are saying the file is licensed under CC0 public domain, but still have a copyright header at the top. The idea of the public domain is to waive copyrights, so if you are going to use that, there should be no copyright statement.